addons/towing/functions/fnc_startTow.sqf

```diff
@@ -17,6 +17,7 @@
  * Public: No
  */
 params ["_unit", "_target", "_ropeClass"];
+[QGVAR(ropeDeployed), [_unit, _target, _ropeClass]] call CBA_fnc_localEvent;
```

Comment:
Should be moved at EOF. The event is named in past tense, but currently happens before anything else.

Reply:
done